src/constants/notification-content.js

```diff
@@ -0,0 +1,167 @@
+export default function getNotificationContent({ type, payload }) {
```

Comment:
Don't see how a function belongs to "constants". Perhaps move it to utils?

Reply:
Well, it specifies the strings of notifications which are more or less constant. I would search for those strings in `constants` first, that's why